client/state/data-layer/wpcom/signup/verticals/index.js

```diff
@@ -26,7 +28,7 @@ export const requestVerticals = action =>
 			path: '/verticals',
 			query: {
 				search: action.search.trim(),
-				site_type: action.siteTypeId,
+				...( action.siteTypeId ? { site_type: action.siteTypeId } : {} ),
```

Comment:
Do you need a ternary here?

```
...( action.siteTypeId && { site_type: action.siteTypeId } )
```

I don't think the property will be added at all during compilation using the above. Not a biggie.

Reply:
I like it, this is my new favourite way to optionally include object properties 👍